public/script.js

```diff
@@ -857,12 +857,12 @@ async function firstLoadInit() {
         throw new Error('Initialization failed');
     }
 
-    await getSystemMessages();
-    sendSystemMessage(system_message_types.WELCOME);
     await getClientVersion();
     await readSecretState();
-    await getSettings();
     initLocales();
+    await getSystemMessages();
```

Comment:
I'll have to see if this loading order actually works for all scenarios. The current order was learned the hard way.

Reply:
Okay, I see the issue now. Fixed it.